content/_policy/rules-of-use._en.md

```diff
@@ -80,7 +80,7 @@ If we suspect fraud on your account, we may lock your account and contact you. I
 
 ## 4. Identity Proofing and Verification
```

Comment:
Minor: When comparing for all differences between this page and the approved document, I noticed in the third paragraph of General Service Definition above, "which you can revoke that at any time" should be "which you can revoke at any time".

Reply:
@aduth let's open a ticket for this as a task to review in upcoming sprints.